en/mkdocs.yml

```diff
@@ -1471,6 +1471,7 @@ nav:
                         - Tuning the VFS Transport: install-and-setup/setup/mi-setup/performance_tuning/tuning-the-vfs-transport.md
                         - Tuning the RabbitMQ Transport: install-and-setup/setup/mi-setup/performance_tuning/rabbitmq_transport_tuning.md
                         - Tuning the Inbound Endpoints: install-and-setup/setup/mi-setup/performance_tuning/tuning-inbound-endpoints.md
+            - WSO2 Installation Options: install-and-setup/setup/installation-options.md
```

Comment:
How come this is under the "Deploy" section and not under "Install"? Wouldn't this be part of standard installation as well?

Also let's change the title to "Installation Options" or "Installation Options for Deployments" (if it is production deployment specific).

I expect this to be higher up in the list of topics. It seems a bit too low down the list at the moment. WDYT?

Reply:
Moved to install section from https://github.com/wso2/docs-apim/pull/4524/commits/cb8da54656fd5ee2779a0a5ac3c98ab0b896b519